docusaurus/docs/e2e-test-a-plugin/use-the-api.md

```diff
@@ -0,0 +1,62 @@
+---
+id: api
+title: The API
+description: How to use the grafana/plugin-e2e API
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - e2e
+  - end-to-end
+  - API
+sidebar_position: 10
+---
+
+`@grafana/plugin-e2e` uses the [page object model](https://playwright.dev/docs/pom) pattern to simplify the authoring of tests and ease maintenance of the codebase. In the page object model, each web page of an application is represented as a class file.
+
+## Classes
+
+In the `@grafana/plugin-e2e` package, classes represent pages or components in Grafana. Their purpose is to encapsulate common UI operations in one place. They also handle UI deviations between different versions of Grafana.
+
+The package exports classes, but the classes are also exposed through the Playwright API via so called [fixtures](https://playwright.dev/docs/test-fixtures).
+
+## Fixtures
+
+The `@grafana/plugin-e2e` package defines a set of [custom fixtures](https://github.com/grafana/plugin-tools/blob/main/packages/plugin-e2e/src/types.ts/#L21-L258) that facilitate the end-to-end testing of Grafana plugins. 
```

Comment:
Not blocking - _maybe_ better to permalink? Either way we need to keep an eye out for when the source changes :shrug:  https://github.com/grafana/plugin-tools/blob/c50ba37b707ba9a561beb8549e89d53d0a4739f0/packages/plugin-e2e/src/types.ts#L21-L258

Reply:
Yeah this is tricky. Would have been amazing if we could generate markdown docs from these tsdocs so we could reference that page instead of the code. But turns out this is pretty tricky. Removing the line references from the permalink for now.